package.json

```diff
@@ -4,11 +4,11 @@
   "description": "OnRack Http Server",
   "main": "index.js",
   "scripts": {
-    "doc": "./node_modules/.bin/jsdoc lib -r -d docs",
+    "doc": "./node_modules/.bin/jsdoc lib/api/1.1/northbound -r -d docs",
     "test": "./node_modules/.bin/mocha $(find spec -name '*-spec.js') -R spec --require spec/helper.js",
     "install": "./install-web-ui.sh",
-    "apidoc": "./node_modules/.bin/apidoc -i lib -f '.*\\.js$' -o build/apidoc",
-    "apidoc-swagger": "./node_modules/.bin/apidoc-swagger --verbose -i lib -f '.*\\.js$' -o build/apidoc-swagger"
+    "apidoc": "./node_modules/.bin/apidoc -i lib/api/1.1/northbound -f '.*\\.js$' -o build/apidoc",
+    "apidoc-swagger": "./node_modules/.bin/apidoc-swagger --verbose -i lib/api/1.1/northbound -f '.*\\.js$' -o build/apidoc-swagger"
```

Comment:
How are developers supposed to access the southbound api and use it? The documentation is needed...